stable/rook-ceph/templates/resources.yaml

```diff
@@ -0,0 +1,699 @@
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: cephclusters.ceph.rook.io
+spec:
+  group: ceph.rook.io
+  names:
+    kind: CephCluster
+    listKind: CephClusterList
+    plural: cephclusters
+    singular: cephcluster
+  scope: Namespaced
+  version: v1
```

Comment:
this should be 

```yaml
  versions:
  - name:  v1
```

instead

Reply:
AFAICS, 'version' has been deprecated, but not removed. Due to Rook needing to remain compatible with k8s 1.15 for a few months, I don't think we can change this right now. (We can push for this in 1.5 though.)

Should this linting error really just be a warning?

Reply:
( https://v1-17.docs.kubernetes.io/docs/setup/release/notes/, claims that rbac.authorization.k8s.io/v1beta1 will no longer be served in v1.20. So this will need to be addressed by upstream rook in the near future.)

Reply:
... and https://kubernetes.io/docs/setup/release/notes/ mentions that authentication.k8s.io/v1beta1 and authorization.k8s.io/v1beta1 will be dropped in 1.22.